product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -0,0 +1,211 @@
+
+* **Create data**
+  The simplest way to test that the cluster is replicating is to log in to a node, create a table, populate it, and see the data you populated appear on a second node. On the first node, 
+    * Create a table:
+      ```sql
+      CREATE TABLE quicktest ( id SERIAL PRIMARY KEY, value INT ); 
+      ```
+    * Populate the table:
+    ```sql
+    INSERT INTO quicktest (value) SELECT random()*10000
+    FROM generate_series(1,10000);
+    ```
+    * Monitor replication performance:
+    ```sql
+    select * from bdr.node_replication_rates;
+    ```
+    * Get a sum of the value column (for checking):
+    ```sql
+    select COUNT(*),SUM(value) from quicktest;
+    ```
+
+* **Check data**
+   * To confirm the data was successfully replicated, log in to a second node.
+     * Get a sum of the value column (for checking):
+      ```sql
+      select COUNT(*),SUM(value) from quicktest;
+      ```
+     * Compare with the result from first node.
+   * Log in to a third node. 
+     * get a sum of the value column (for checking):
+      ```sql
+      select COUNT(*),SUM(value) from quicktest;
+      ```
+   * Compare with the result from the first and second nodes.
+
+## Worked example
+
+### Preparation
+
+Our cluster has three data nodes, kaboom, kaftan and kaolin. We will use kaboom as our first node. Log in to kaboom and then into kaboom's Postgres server. 
+
+```
+cd democluster
+ssh -F ssh_config kaboom
+sudo -iu enterprisedb psql bdrdb
+```
+
+#### Ensure the cluster is ready
+
+To ensure that the cluster is ready to go, run: 
+
+```
+select bdr.wait_slot_confirm_lsn(NULL, NULL);
+__OUTPUT__
+ wait_slot_confirm_lsn
+-----------------------
+
+(1 row)
+```
+
+This query waits if the cluster is busy initializing and returns when the cluster is ready.
+
+### Create data
+
+#### On the first node (kaboom), create a table
+
+Run:
+
+```sql
+CREATE TABLE quicktest ( id SERIAL PRIMARY KEY, value INT ); 
+__OUTPUT__
+CREATE TABLE
+```
+
+#### On kaboom, populate the table
+
+```
+INSERT INTO quicktest (value) SELECT random()*10000 FROM generate_series(1,10000);
+__OUTPUT__
+INSERT 0 10000
+```
+
+This command generates a table of 10000 rows of random values. 
+
+#### On kaboom, monitor performance
+
+As soon as possible, run:
+
+```sql
+select * from bdr.node_replication_rates;
+__OUTPUT__
+ peer_node_id | target_name | sent_lsn  | replay_lsn | replay_lag | replay_lag_bytes | replay_lag_size | apply_rate | catchup_interval
+--------------+-------------+-----------+------------+------------+------------------+-----------------+------------+------------------
+   3490219809 | kaftan      | 0/F57D120 | 0/F57D120  | 00:00:00   |                0 | 0 bytes         |       9158 | 00:00:00
+   2111777360 | kaolin      | 0/F57D120 | 0/F57D120  | 00:00:00   |                0 | 0 bytes         |       9293 | 00:00:00
+(2 rows)
+```
+
+The command shows statistics about how quickly that data was replicated to the other two nodes:
```

Comment:
Should this just end with a period or is something missing? Again, would it be more useful to put before the code, i.e.,

As soon as possible, run the following command. The command shows statistics about how quickly that data was replicated to the other two nodes.